src/lein_monolith/dependency.clj

```diff
@@ -193,7 +193,7 @@
                  (count (distinct (map dep-source specs)))
                  " projects - using " (pr-str default-choice) " from "
                  (dep-source default-choice))
-            (ansi/sgr :red)
+            (ansi/maybe-sgr :red)
```

Comment:
To simplify things, I think we can just remove the coloring here. That way only tasks depend directly on the color behavior.